### ISSUE TITLE
Country/profession color-coding for households

### ISSUE THREAD

**Issue by tammyalexandra** [skeetidot/homestead-hebrews#12]
For the dots which collect multiple addresses for people living in the same households, the color-coding for country of origin and profession needs some tweaks.  

For ex -- 1908 -- 406 Dickson street has my ggf living with two other people.  The dot is white (other/unknown) -- but I know my ggf was born in Hungary, even if you don't have data for the other two.  Or -- also 1908 -- the Lebovitz household is blue (USA) -- but this isn't the right thing to show, bc altho some of the children were born in the U.S., the head of household was born in Hungary, so it makes more sense to think of this as a Hungarian, rather than an American, household.

I'm assuming the same challenge is happening with profession, although I didn't look.

Fixing this may be super-complicated since it involves understanding who is the head of a particular household.  Let's discuss?

**Comment by tammyalexandra** [comment]
Also noticed while cleaning up duplicates as per #1 that some people have multiple professions.

**Comment by skeetidot** [comment]
I agree with you that this would involve identifying the head of household for each address and year and using their profession and birth country to determine the color.

All the dots are actually there, but they are drawing on top of each other, so you can't see the other values. I think this would require looping through all the dots and moving the head of household dots above the others.

**Comment by tammyalexandra** [comment]
Or maybe I need to get smarter about how I produce the feed?
